assets/stylesheets/sections/_sharing.scss

```diff
@@ -249,6 +249,11 @@ $color-print: #f8f8f8;
 			border-color: lighten( $alert-yellow, 8% ) !important;
 		}
 	}
+
+	&.is-compact{
```

Comment:
Nothing, it looks like an error. I cannot for the life of me figure out why it's there. Will remove.